docs/en_US/Release.md

```diff
@@ -1,5 +1,49 @@
 # ChangeLog
 
+# Release 1.9 - 10/22/2020
+
+## Major updates
+
+### Neural architecture search
+* Support regularized evolution algorithm for NAS scenario (#2802)
+* Add NASBench201 in search space zoo (#2766)
+
+### Model compression
+* AMC pruner improvement: support resnet, support reproduction of the experiments (default parameters in our example code) in AMC paper (#2876 #2906)
+* Support constraint-aware on some of our pruners to improve model compression efficiency (#2657)
+* Support "tf.keras.Sequential" in model compression for TensorFlow (#2887)
+* Support customized op in the model flops counter (#2795)
+* Support quantizing bias in QAT quantizer (#2914)
+
+### Training service
+
+* Support of configuring python environment using "preCommand" in remote mode (#2875)
+* Support AML training service in Windows (#2882)
+* Support reuse mode for remote training service (#2923)
+
+### WebUI & nnictl
+
+* Support the new design of the Overview page. (#2914)
+* Upgraded node, yarn and FabricUI, and enabled Eslint
+* Add/Remove columns in hyper-parameter chart and trial details table and Refactor trial details table (click event) (#2900)
```

Comment:
Remove the refactor part.

Or at least write `refactor` instead of `Refactor`.

Reply:
removed